src/Uno.UI/UI/Xaml/Shapes/Shape.cs

```diff
@@ -93,7 +90,7 @@ public double StrokeThickness
 			typeof(double),
 			typeof(Shape),
 			new FrameworkPropertyMetadata(
-				defaultValue: 1.0d,
+				defaultValue: 0d,
```

Comment:
This isn't the correct fix, 1 is indeed the default value on UWP (even though the docs state otherwise)

Reply:
The majority of these changes should still fix the related issues, but I will change this back to 1.0 and remove the IsDependencyPropertySet logic for the ActualStrokeThickness property